packages/skills/carpark_client/handlers.py

```diff
@@ -218,7 +220,8 @@ def _handle_match_accept(self, msg: FIPAMessage, sender: str, message_id: int, d
                                     sender_tx_fee=0,
                                     counterparty_tx_fee=0,
                                     quantities_by_good_pbk={},
-                                    dialogue_label=dialogue.dialogue_label.json)
+                                    dialogue_label=dialogue.dialogue_label.json,
+                                    ledger_id='fetchai')
```

Comment:
Have a look at my PR to see how I move all constants to configs or load from aea.

Reply:
Do you mean things like this (from your latest PR):
    sender_tx_fee=strategy.max_buyer_tx_fee,

And do you envisage this being applied to:
    sender_tx_fee
    counterparty_tx_fee
    ledger_id

and specifically this function or were you talking more generally in the code?

Reply:
I was referring to `fetchai` constant which I think should be provided via an instance variable in the strategy (or some other class of your choice) and loaded from config. That way it is trivial to also run the carpark agent on other ledgers.

Reply:
It might indeed help to apply it to `sender_tx_fee`, `counterpartt_tx_fee`, `ledger_id` and `currency_pbk` :)

Reply:
Hi,

I am not going to have time to fix this prior to 3:30 which is when I need to get things ready by.